application/features/user.feature

```diff
@@ -29,7 +29,7 @@ Feature: User
         | locked       | no                    |
       And the following data is not returned:
         | property                |
-        | password_hash           |
+        | password_id             |
         | password_expires_at_utc |
```

Comment:
yes, working out the tests on that stuff is going to have to wait since it may not be trivial.